### DISCUSSION TITLE
Plotting of signal on the chart

### DISCUSSION THREAD

**Discussion by sultank1** [kernc/backtesting.py#683] [General]
Hi kernc...
Is there a way to  plot on the chart, the signal type(long/short)?

**Comment by kernc** [comment]
Can you share a bit of your code? Normally, every indicator ("signal") wrapped in [`Strategy.I()`](https://kernc.github.io/backtesting.py/doc/backtesting/backtesting.html#backtesting.backtesting.Strategy.I) would be plotted automatically.

And see https://github.com/kernc/backtesting.py/issues/87#issuecomment-643833897 for a case of plotting a signal that is unavailable at `init()` but computed in `next()`.

**Comment by sultank1** [comment]
Sorry I must have not explained clearly...

is there way to show the direction(BUY, SELL)  of the trade on the chart.
currently it only indicate profit or loss of each trade on the price
line...

But I guess it won't be necessary as I can easily calculate that based on
if trade resulted in profit or loss.

Thanks by the way...

**Reply by kernc** [reply]
See https://github.com/kernc/backtesting.py/issues/676#issuecomment-1172283308. The triangles' direction (up/down) respectively indicates long/short trades.

**Reply by sultank1** [reply]
Thanks. 
😅 I guess I should have thoroughly read the documentation...
